app/models/planet.py

```diff
@@ -0,0 +1,26 @@
+from app import db
+
+class Planet(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String, nullable=False)
+    description = db.Column(db.String)
+    color = db.Column(db.String)
+    moon = db.relationship("Moon", back_populates="planet")
```

Comment:
small change, but a planet can have multiple moons. Maybe call this field `moons` instead.